internal/services/mssqlmanagedinstance/mssql_managed_instance_resource.go

```diff
@@ -653,3 +609,32 @@ func (r MsSqlManagedInstanceResource) normalizeSku(sku string) string {
 
 	return sku
 }
+
+// the StorageAccountType property has changed to RequestedBackupStorageRedundancy with 1-1 mapping of the following values:
+// GRS -> Geo
+// ZRS -> Zone
+// LRS -> Local
+// GZRS -> GeoZone
+func storageAccTypeToBackupStorageRedundancy(storageAccountType string) managedinstances.BackupStorageRedundancy {
+	switch storageAccountType {
+	case string(sql.StorageAccountTypeZRS):
+		return managedinstances.BackupStorageRedundancyZone
+	case string(sql.StorageAccountTypeLRS):
+		return managedinstances.BackupStorageRedundancyLocal
+	case "GZRS":
+		return managedinstances.BackupStorageRedundancyGeoZone
+	}
+	return managedinstances.BackupStorageRedundancyGeo
```

Comment:
Can we define local constants/enums for these? Seems a lot of overhead to import the `sql` package for them?